sdk/storage/perf-test/src/main/java/com/azure/storage/blob/perf/UploadFromFileTest.java

```diff
@@ -3,33 +3,31 @@
 
 package com.azure.storage.blob.perf;
 
+import static com.azure.perf.test.core.TestDataCreationHelper.createRandomInputStream;
+
+import com.azure.perf.test.core.PerfStressOptions;
+import com.azure.storage.blob.perf.core.BlobTestBase;
 import java.io.FileOutputStream;
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.OutputStream;
 import java.nio.file.Files;
 import java.nio.file.Path;
-
-import com.azure.perf.test.core.RandomStream;
-import com.azure.perf.test.core.SizeOptions;
-import com.azure.storage.blob.perf.core.BlobTestBase;
 import reactor.core.publisher.Mono;
 
-public class UploadFromFileTest extends BlobTestBase<SizeOptions> {
+public class UploadFromFileTest extends BlobTestBase<PerfStressOptions> {
+
+    private final Path tempFile;
```

Comment:
Why move from static to instance?  All instances of this test should be able to upload from the same file on disk, right?

Reply:
Either way would work, but my preference is to share the same file unless each test needs its own file.  It may also be slightly faster since the OS disk cache only needs to cache one file, not N files.

Reply:
Moving this from static to instance actually breaks the test, since the file is only created and deleted in `globalSetup`, not per-instance `setup`.

Reply:
Yes, I was about to update globalSetup to per instance setup but yeah, it makes sense to have a single file. I will switch this back to static.